tools/iothub-explorer/iothub-explorer.js

```diff
@@ -20,6 +20,7 @@ var ConnectionString = require('azure-iothub').ConnectionString;
 var SharedAccessSignature = require('azure-iothub').SharedAccessSignature;
 var anHourFromNow = require('azure-iot-common').anHourFromNow;
 var DeviceSAS = require('azure-iot-device').SharedAccessSignature;
+var Util = require('util');
```

Comment:
I may be mistaking this for something else, but if this is the native util package, our convention is to use lowercase for the variable name